docs/user-guide/expressions/numpy.md

```diff
@@ -1,7 +1,7 @@
-# Numpy
+# Numpy ufuncs
 
 Polars expressions support NumPy [ufuncs](https://numpy.org/doc/stable/reference/ufuncs.html). See [here](https://numpy.org/doc/stable/reference/ufuncs.html#available-ufuncs)
-for a list on all supported numpy functions.
+for a list on all supported numpy functions. Additionally, SciPy offers a wide host of ufuncs. Specifically, the [scipy.special](https://docs.scipy.org/doc/scipy/reference/special.html#module-scipy.special) namespace has ufunc versions of many (possibly most) of what is available under stats. 
```

Comment:
_so_ glad you've opened a PR about this 🙌 

I think the lint failure is just because you have a trailing space at the end of this line